connectors/spark/src/main/scala/io/unitycatalog/spark/UCHadoopConf.java

```diff
@@ -42,4 +42,16 @@ private UCHadoopConf() {
   public static final String UC_CREDENTIAL_CACHE_ENABLED_KEY =
       "unitycatalog.credential.cache.enabled";
   public static final boolean UC_CREDENTIAL_CACHE_ENABLED_DEFAULT_VALUE = true;
+
+  // Keys for retry configuration.
+  public static final String RETRY_MAX_ATTEMPTS_KEY = "unitycatalog.retry.max.attempts";
+  public static final int RETRY_MAX_ATTEMPTS_DEFAULT = 4;
+
+  public static final String RETRY_INITIAL_DELAY_KEY = "unitycatalog.retry.initial.delay.ms";
```

Comment:
Please follow the guidelines, about the configure keys naming: https://github.com/unitycatalog/unitycatalog/pull/1108#discussion_r2427004787